internal/strict/controls/deprecated_command.go

```diff
@@ -7,16 +7,27 @@ import (
 )
 
 const (
-	DefaultCommandsCategoryName = "Default commands"
-	RunAllCommandsCategoryName  = "`*-all` commands"
+	DefaultCommandsCategoryName     = "Default commands"
```

Comment:
DefaultCommandsCategoryName is not in use

Reply:
Since I cut all these PRs from the final code where all commands were deprecated, I forgot to remove them, they are used in other PRs. If I remove them from this branch now, I'll have to insert them into other branches, but since they will be merged at the same time, it makes sense to turn a blind eye to it.